### PR TITLE
Fix typo and golangci

### PR DESCRIPTION
**Pull request by futuretea**
- Fix typo
- Remove deprecated linter deadcode (Replaced by unused)

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -10,7 +10,6 @@ linters:
     - gofmt
     - goimports
     - whitespace
-    - deadcode
```
Comment (masteryyh) [inline .golangci.yml:13]: Why disabling `deadcode`?

Comment (futuretea) [reply .golangci.yml:13]: included in linter `unused`

**Review by masteryyh — APPROVED** [review]
LGTM, thanks

**Review by FrankYang0529 — APPROVED** [review]
LGTM

**Pull request merged**